### ISSUE TITLE
does it work in termux

### ISSUE THREAD

**Issue by foxyisnotsalt** [urbanadventurer/Android-PIN-Bruteforce#123]

**Comment by ThePieMonster** [comment]
Add the following at the top of the script file `android-pin-bruteforce`. 
`#!/data/data/com.termux/files/usr/bin/bash`

**Comment by urbanadventurer** [comment]
Did this solution help?

**Comment by vsa77** [comment]
Yes. I had it working in Termux on a rooted Moto G7 Play in 2021. Don't recall if I needed to do the modification listed above, but if you run into errors, try it.

Recommend closing this issue and possibly adding a section to the readme that lists devices/builds (i.e. Nethunter, Termux, etc) it's been verified to run on.
